mmseg/models/backbones/swin.py

```diff
@@ -708,14 +708,9 @@ def init_weights(self):
                 self.pretrained, logger=logger, map_location='cpu')
             if 'state_dict' in ckpt:
                 state_dict = ckpt['state_dict']
-            elif 'model' in ckpt:
-                state_dict = ckpt['model']
             else:
                 state_dict = ckpt
 
-            if self.pretrain_style == 'official':
-                state_dict = swin_convert(state_dict)
-
             # strip prefix of state_dict
```

Comment:
The `pretrain_style` argument is not used, so it can be removed from `__init__`.

Reply:
Handled in #800